espanol/dragon/dragon.md

```diff
@@ -1,5 +1,5 @@
 Para iniciar la batalla, tenía que conseguir un medio de transporte.
-Y este medio de transporte sería el Dragón Earhart, que compró al más famoso
+Y este medio de transporte sería el Dragón Earhart, que compró al másfamoso 
```

Comment:
That's no problem! If you fix it, I'm still happy to merge the pull request. Make a new commit adding the space back, push your change to the same branch you used to create this pull request, and re-open the pull request.